README.md

```diff
@@ -55,6 +55,7 @@ To contribute, fork this repository, add your new resource and submit a PR. For
 ## Extensions & Plugins
 
 * [Alpine.js IntelliSense - Extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=adrianwilczynski.alpine-js-intellisense)
+* [Spruce a lightweight state management layer for Alpine.js](https://github.com/ryangjchandler/spruce)
```

Comment:
"Spruce - a lightweight state management layer"

I don't think we need the Alpine.js bit here either (I'm the project owner so I can say 😉)

Reply:
done 👍